doc/api/configuration.md

```diff
@@ -105,6 +105,46 @@ Automatically limit the size of the returned result sets (e.g., add a `LIMIT` at
 %config SqlMagic.autolimit = 0
 ```
 
+## `column_local_vars`
+
+Default: `False`
+
+Return the data stored in each column
+of the result as a local variable.
+
+```{code-cell} ipython3
+%config SqlMagic.column_local_vars = True
+%sql SELECT * FROM languages LIMIT 2
+```
+
+The data of each column is now stored as
+a list with variable name of the column.
+
+```{code-cell} ipython3
+print(name)
+print(rating)
+```
+
+```{code-cell} ipython3
+%config SqlMagic.column_local_vars = False
+```
+
+
+## `autocommit`
+
+Default: `True`
+
+Commit each executed query to the
```

Comment:
It's not clear when this will be useful. 
There was some refactoring done as part of this [PR](https://github.com/ploomber/jupysql/pull/768/files).
Please add if any special case needs to be mentioned